router/client/src/client.rs

```diff
@@ -106,45 +107,74 @@ impl Client {
         max_total_tokens: u32,
     ) -> Result<Option<u32>> {
         let mut n_tokens = 0;
+        let mut req_id = 0;
         let mut requests = Vec::new();
+        let skip_tokenizer_in_tgi = env::var("SKIP_TOKENIZER_IN_TGI").ok().map_or(false, |value| value.to_lowercase() == "true");
         // Create requests
         while n_tokens < max_prefill_tokens {
             let truncate = min(max_input_length, max_prefill_tokens - n_tokens);
+            let inputs = if skip_tokenizer_in_tgi {
+                "1, 1, 518, 25580, 29962, 3532, 14816, 29903, 6778, 13, 3492, 526, 385, 319, 29902, 
```

Comment:
What are these values? Why were they chosen? Note that different models have different tokenizers.

Reply:
It should not be hardcoded, warmup should contain random data